File: orm-spring-mongodb/src/main/java/com/example/ormspringmongodb/model/Student.java
package com.example.ormspringmongodb.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.Date;

@Document("students")
@Data @AllArgsConstructor @NoArgsConstructor @Builder
public class Student {

    @Id
    private Long Id;
    private String fullName;
    private String email;
    private Date dateStart;
    private double note;
}

File: AspOrmMongo/AspOrmMongo/Model/Student.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace AspOrmMongo.Model
{
    public class Student
    {
        [BsonId]
        public ObjectId Id { get; set; }
        public string FullName { get; set; }
        public string Email { get; set; }
        public DateTime DateStart { get; set; }
        public double Note { get; set; }
    }
}
